llabake/Hello-Books
# bug(NavBar-Footer): Fix Nav Bar and Jumping Footer

#### What does this PR do?
This PR fixes the application footer and navigation bar component

#### Description of Task to be completed?
fix jumping footer
style to footer component
separate navbar based on role and authentication
change existing navigation bar in components

#### How should this be manually tested?
clone the repo
cd into the project directory
run` npm install` to install all the dependency
run `npm run start-dev` to start the back-end server
run `npm run dev` to start the front-end server
click on the user dropdown you should see sign up and sign in if you are unauthenticated, all pages list in the drop-down if you are an admin and  all pages list except the admin dashboard and add book link if you are a normal user

#### What are the relevant pivotal tracker stories?
157534616

#### Any background context you want to add?
N/A

#### Screenshots
N/A
